pkg/ublox/ublox.go

```diff
@@ -226,19 +250,70 @@ func match(stdout string, ubLoxRegex *regexp.Regexp) (string, error) {
 	return "", fmt.Errorf("error parsing %s", stdout)
 }
 
-// GetNavOffset ... get gnss offset
 func (u *UBlox) GetNavOffset() (string, error) {
-	args := []string{"-t", "-p", "NAV-CLOCK", "-P", "29.20"}
 
-	output, err := exec.Command(UBXCommand, args...).Output()
-	if err != nil {
-		return "", fmt.Errorf("error executing ubxtool command: %s", err)
-	}
+	output, _ := u.navPoll.UbloxPoll(1000000000, "tAcc")
```

Comment:
use `0` to wait forever.

Reply:
Support for that (and also for larger numbers) was broken when I tested.  This should wait about 30 years so would be sufficient.